generate.php

```diff
@@ -10,17 +10,18 @@
 
 $client = new \Github\Client();
 $token = $argv[1];
-$client->authenticate($token, null, Github\Client::AUTH_HTTP_TOKEN);
+$client->authenticate($token, null, Github\Client::AUTH_ACCESS_TOKEN);
 
 $branchManager = new \App\PrestaShopModulesReleaseMonitor\BranchManager($client);
 
 function getModules($client): array
 {
+    $excludedRepos = ['gamification', 'ps_emailsmanager', 'welcome'];
```

Comment:
Hello @okom3pom I'm so sorry for coming back after all this time. I've got some time now :)

PR seems OK but I don't understand why you exclude these 3 repositories. Can you tell me?

Reply:
Hello, these 3 repositories are from PrestaShop Corp.

Reply:
Welcome module is on the OSS project organization https://github.com/PrestaShop/welcome

Reply:
Hmm, that's strange. This module is not used anymore in PrestaShop 8.0, I correct the PR

Reply:
Autoupgrade module is also not embedded in the software ;) but available on download